PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -20,7 +20,7 @@ class PurchasesTests: XCTestCase {
             "other_purchases": [:],
             "original_application_version": NSNull()
     ]]
-    
+
```

Comment:
Xcode's formatter got angry at all of these. happy to revert if folks find it annoying / irrelevant to the PR. but I figured we might as well just get them all fixed in one pass.

Reply:
I like this.